app/src/main/cpp/droidvnc-ng.c

```diff
@@ -103,6 +103,30 @@ static void onKeyEvent(rfbBool down, rfbKeySym key, rfbClientPtr cl)
     (*theVM)->DetachCurrentThread(theVM);
 }
 
+/*
+ * Is equivalent to calling Charset.forName("ISO-8859-1").decode(byteBuffer).toString()
+ */
+static jstring stringDecode(JNIEnv *env, const char* text)
```

Comment:
Can you please inline that code into onCutText()?

Reply:
of course